src/components/tasks/card/index.tsx

```diff
@@ -9,7 +9,7 @@ const moment = require('moment');
 type Props = {
   content: task;
   shouldEdit: boolean;
-  onContentChange: any;
+  onContentChange: (changeId: string, changeObject: object) => void;

```

Comment:
It's capital O right?
Didn't it throw an error?

Reply:
it's not capital O

Reply:
Oh yes it is type, sorry for the comment